src/components/OrderItemActionsPopover.vue

```diff
@@ -1,21 +1,32 @@
 <template>
   <ion-content>
     <ion-list>
-      <ion-list-header>{{ "primary identifier" }}</ion-list-header>
-      <ion-item>
+      <ion-list-header>{{ getProductIdentificationValue(productIdentificationStore.getProductIdentificationPref.primaryId, getProduct(item.productId)) || getProduct(item.productId).productName }}</ion-list-header>
```

Comment:
In case of opening popover for header, we are not passing item, so need to handle that case in this component.